src/cogent3/core/new_alignment.py

```diff
@@ -2647,16 +2759,23 @@ def get_seq_array(
         """Return sequence corresponding to seqid as an array of indices.
         start/stop are in alignment coordinates. Excludes gaps.
         """
-        seq = self.seqs[seqid]
-        gaps = self.gaps[seqid]
-
+        seq = self._seqs[seqid]
```

Comment:
add "and the alignment coordinates are converted to sequence coordinates" in the docstring